### PR TITLE
Auth 137 move operator to the openshift prefixed namespace

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/AUTH-137

This Pull Request moves all operands from `cert-manager` into `openshift-cert-manager` namespace.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,7 +71,7 @@ verify: verify-scripts verify-codegen-crds
 
 local-deploy-manifests:
 	kubectl apply -f ./manifests
-	kubectl apply -f ./bundle/manifests
+	kubectl apply $$(ls ./bundle/manifests/*.crd.yaml | awk ' { print " -f " $$1 } ')
```
Comment (s-urbaniak) [inline Makefile:74]: suggestion: maybe let's put the cluster service version or the CRDs in its own directory?

Comment (slaskawi) [reply Makefile:74]: We'd need to move it around in the downstream then. It's not nice that CRDs and expected to coexist with CSVs in bundles :face_exhaling:

Comment (s-urbaniak) [reply Makefile:74]: ah ... ugh, that's a strange constraint.

**Pull request merged**